backend/inspirehep/orcid/utils.py

```diff
@@ -226,3 +233,37 @@ def distributed_lock(lock_name, expire=10, auto_renewal=True, blocking=False):
 
 class DistributedLockError(Exception):
     pass
+
+
+def update_moved_orcid(old_orcid, new_orcid):
+    try:
+        author_record = AuthorsRecord.get_record_by_pid_value(old_orcid, "orcid")
+        if new_orcid not in get_value(author_record, "ids.value"):
+            new_author_ids = [
+                {"schema": "ORCID", "value": new_orcid},
+                *author_record["ids"],
+            ]
+            author_record["ids"] = new_author_ids
+            author_record.update(dict(author_record))
+    except PIDDoesNotExistError:
+        LOGGER.info(
```

Comment:
I don't think this should be logged with such a low log level. If we're trying to push to this ORCID, we necessarily need to have an author profile and account corresponding to this ORCID. So if you can't find it, something is really wrong and we want to know about it.

Reply:
So you mean to raise an exception there?